packages/api/src/webhooks/cannon.ts

```diff
@@ -90,6 +90,10 @@ export default class WebhookCannon {
   }
 
   async processWebhookEvent(event: messages.WebhookEvent): Promise<boolean> {
+    if (event.event === "playback.accessControl") {
+      // Cannot fire this event as a webhook, this is specific to access control and fired there
```

Comment:
Adding some more comment+runtime log info:
```suggestion
      // Cannot fire this webhook asynchronously. This is specific to access control and fired on
      // the /access-control/gate API controller synchronously
      console.error("Received unexpected access control event in async webhook cannon. id=${event.id} userId=${event.userId}")
```